peps/pep-0761.rst

```diff
@@ -207,9 +207,57 @@ During the `pre-PEP discussion <https://discuss.python.org/t/pre-pep-discussion-
 there was a question of whether offline verification was supported by
 Sigstore. Using a Sigstore bundle (:file:`.sigstore`) file, `Sigstore clients
 support verifying the artifact completely offline <https://discuss.python.org/t/pre-pep-discussion-stop-providing-gpg-signatures-for-cpython-artifacts/65058/9>`_.
-When in offline mode, Sigstore can't verify whether a signature has been
-revoked. This is a similar restriction to PGP key revocations not being
-detectable during offline verification.
+
+Using offline verification with Sigstore requires disabling root of trust
+updates and "pinning" a root of trust in a file to use during verification.
+
+Pinning a root of trust means signatures made after a *new* root of trust
+is established would no longer be able to verify using a "pinned" previous
+root of trust. New roots of trust are expected to be rare events, such as
+when the root of trust is compromised, and in this case verifiers would
+want signatures to fail to verify.
+
+Offline verification also makes revocation checks impossible, but this
+is similar to PGP's model where revocation of keys requires an online lookup.
+
+Barring rare events like root of trust compromise, using offline verification
+with Sigstore doesn't impose additional operations requirements to verifiers.
+
+Support for a pre-compiled executable for verification
+------------------------------------------------------
+
+During discussion there were requests for a pre-compiled executable that could
+be used for verifying Sigstore bundles without needing to either install
+a Go build toolchain to build `sigstore-go <https://github.com/sigstore/sigstore-go>`_
+from source or already have a working Python installation for
+`sigstore-python <https://github.com/sigstore/sigstore-python/>`_.
+
+`Cosign <https://github.com/sigstore/cosign/>`_ is another Sigstore project
+that provides pre-compiled standalone binaries and supports verifying bundles
+offline:
+
+ .. code-block::
```

Comment:
```suggestion
.. code-block::
```

The single-space indent causes it to be blockquoted:

<img width="784" alt="image" src="https://github.com/user-attachments/assets/d9ca6a81-b608-4e71-89fb-b88bffaf563f">

Without:

<img width="809" alt="image" src="https://github.com/user-attachments/assets/717bf59e-2a3a-4564-be70-9bd0b065ea02">